theia-docker/Dockerfile

```diff
@@ -28,7 +29,8 @@ ENV HOME /home/theia
 WORKDIR /home/theia
 COPY --from=0 --chown=theia:theia /home/theia /home/theia
 EXPOSE 3000
-ENV SHELL /bin/bash
+ENV SHELL=/bin/bash \
+    THEIA_DEFAULT_PLUGINS=local-dir:/home/theia/plugins
```

Comment:
Should we document somewhere that this is the folder where a user can add VS Code extensions on their own? In application-level README?

Reply:
> Should we document somewhere that this is the folder where a user can add VS Code extensions on their own? In application-level README?

Since all images will eventually be updated to include plugin support (with builtins), should we document it globally (root of the repo), or per application?

Reply:
However I am not sure we'll update all the images in the repo to use plugins. Or at least some will be low-priority and probably be "PR welcome" situation.

Reply:
> However I am not sure we'll update all the images in the repo to use plugins. Or at least some will be low-priority and probably be "PR welcome" situation.

@marcdumais-work I think all images at some point will need to be updated to make use of the builtins and remove deprecated extensions, but you are right that it can be done by other members of the community. Should I update the `readme` for each image I touch? I think some images themselves do not have readmes so they should align. I can also do a pass later if you prefer and update the readmes of the images and add new ones if none exist.

Reply:
I think it will be ideal to have a README for each image. But for now you can add this info in the main README.

Reply:
> I think it will be ideal to have a README for each image. But for now you can add this info in the main README.

I'll do the updates as part of https://github.com/theia-ide/theia-apps/pull/279